### PR TITLE
1890 renaming symbols with expanded iris

### LINKED ISSUE #1890: renaming symbols using expanded IRIs
Moreover, trying to rename a symbol fails:
```
ontology O = 
 DOLCE-Lite with particular |-> Particular
```
gives an error that the symbol `particular` is not known in the signature, although the prefixed symbol is there (last line of the signature fragment I included). The same error occurs for
`http://www.loa-cnr.it/ontologies/DOLCE-Lite.owl#particular |-> Particular` and
`<http://www.loa-cnr.it/ontologies/DOLCE-Lite.owl#particular> |-> Particular`.
```
*** Error:
unknown symbols '[particular]'
file:///home/mcodescu/repos/testSpecs/DOLCE-lite.dol:7.13-7.16:
*** Error:
unknown symbol: particular
in signature:
Prefix: : <http://www.loa-cnr.it/ontologies/DOLCE-Lite.owl#>
Prefix: owl: <http://www.w3.org/2002/07/owl#>
Prefix: rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
Prefix: rdfs: <http://www.w3.org/2000/01/rdf-schema#>
Prefix: xsd: <http://www.w3.org/2001/XMLSchema#>

Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#abstract
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#abstract-quality
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#abstract-region
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#accomplishment
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#achievement
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#amount-of-matter
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#arbitrary-sum
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#dependent-place
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#endurant
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#event
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#feature
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#non-physical-endurant
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#non-physical-object
Class: //www.loa-cnr.it/ontologies/DOLCE-Lite.owl#particular
```

### PR DESCRIPTION
**Pull request by b-gehrke**
Closes #1890

### CODE REVIEW

**Review by mcodescu — APPROVED** [review]
:+1: The tests go through, code looks good.

**Comment by b-gehrke** [comment]
@mcodescu Can you please review the changes since your last review?

File: Common/IRI.hs [hunk 1]
```diff
@@ -114,28 +114,38 @@ For example, for the (full) IRI
...
 -}
 
 
 
 data IRI = IRI
-    { iriScheme :: String         -- ^ @foo:@
+    { iriPos :: Range             -- ^ position
+
+    -- fields used for storing an absolute IRI
+      iriScheme :: String         -- ^ @foo:@
```
Comment (mcodescu) [inline Common/IRI.hs:135]: missing comma

**Review by mcodescu — APPROVED** [review]
:+1:  Well done, I'm glad issues with IRIs are fixed now.

**Pull request merged**